Client/src/Components/Graph/GraphControls/Axes/AxisStateControl.tsx

```diff
@@ -105,27 +106,30 @@ export const AxisStateControl = (props: IProps) => {
 
   return (
     <Grid container spacing={4} justify='space-between' alignItems="center">
-      <Grid item>
-        <FormControl>
+      <Grid item sm={4}>
+        <FormControl fullWidth>
           <InputLabel id={`${axisName}VariableLabel`}>{axisName} Variable</InputLabel>
           <Select
             labelId={`${axisName}Variable`}
+            fullWidth
             id={`${axisName}Variable`}
             value={axisState.variableName}
             autoWidth={true}
             onChange={handleVariableChange}
+            label="Asynchronous"
```

Comment:
i think the label has no effect here does it? what does it do

Reply:
it was for the previous solution, removed it now